sql/migrations/20200417194325_world.sql

```diff
@@ -0,0 +1,110 @@
+
+-- add quest Desperate Prayer (5640) to Priestess Alathea (11401)
+INSERT INTO `creature_questrelation` (`id`, `quest`, `patch_min`, `patch_max`) VALUES
+(11401, 5640, 0, 10);
+
+-- add completion text for Desperate Prayer (5635, 5636, 5637, 5638, 5639)
+UPDATE `quest_template` SET `OfferRewardText` = "I'm glad you've arrived, $n. There are many things we will have to discuss, but more importantly is your training in the ways of the Light.$b$bThere are lessons all servants of the Light must learn. If you are ready, we'll begin discussing some of those now." WHERE `entry` IN (5635, 5636, 5637, 5638, 5639);
+
+-- update completion text for Arcane Feedback (5677)
+UPDATE `quest_template` SET `OfferRewardText` = "I'm glad you've arrived, $n. There are many things we will have to discuss, but more importantly is your training in the ways of the Light.$b$bThere are lessons all servants of the Light must learn. If you are ready, we'll begin discussing some of those now." WHERE `entry` = (5677);
+
+-- add missing quest Arcane Feedback (5678)
+INSERT INTO `quest_template` (`entry`, `ZoneOrSort`, `MinLevel`, `QuestLevel`, `RequiredClasses`, `RequiredRaces`, `QuestFlags`, `ExclusiveGroup`, `Title`, `Details`, `Objectives`, `OfferRewardText`, `RewXP`, `RewMoneyMaxLevel`, `RewSpell`, `CompleteEmote`) VALUES
+(5678, -262, 20, 20, 16, 1, 8, 5676, "Arcane Feedback", "It is time for you to return to Stormwind, young one. Your superiors there have called you back for more training. And although it disturbs me some, your race and mine are allies and I would not betray that.$b$bI do not mean to be so coy with you--my distrust of arcane magics, be they divine inspired or not, is still strong within me.$b$bYou will know of what I speak after you return to your Cathedral of Light. Speak to High Priestess Laurena when you arrive.", `Speak to High Priestess Laurena in Stormwind.`, "Ah, you have wonderful timing, $n. I was beginning to wonder if all of my messengers had been successful. I'm glad to see they were, otherwise I would wager you'd still be out in the world learning more about all the threats that exist against our people.$b$bAre you ready? Ready for more training I mean. You're growing in power more quickly, and I feel you are ready to learn more about our faith.", `390`, `240`, `19345`, `1`);
+
+-- add quest Arcane Feedback (5678) to Priestess Alathea (11401)
+INSERT INTO `creature_questrelation` (`id`, `quest`, `patch_min`, `patch_max`) VALUES

```

Comment:
You have added the quest giver npc, but didn't add who it should be turned in to.